### ISSUE TITLE
Some BE form fields lack proper grouping

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2292]
Related form fields need to share a semantic association. We've tackled this in many places, but a few components still need attention:
- [ ] color picker
- [ ] resource kind picker

**Comment by dananjohnson** [comment]
The resource kind picker should also probably be refactored as radio controls. Currently it's not possible to select them with a keyboard.
